### ISSUE TITLE
Query went into ERROR state due to NPE on TimestampExtractor

### ISSUE THREAD

**Issue by spena** [confluentinc/ksql#5432]
**Describe the bug**

A query went into status ERROR caused by a NPE the processing row has a NULL value as a timestamp:

```
org.apache.kafka.streams.errors.StreamsException: Fatal user code error in TimestampExtractor callback for record ConsumerRecord(topic = REDDI.RWWS.EHSTD.STAGING.ProdLoc, partition = 2, leaderEpoch = null, offset = 15, CreateTime = 1589295072339, serialized key size = 28, serialized value size = 86, headers = RecordHeaders(headers = [], isReadOnly = false), key = Struct{ROWKEY=000000000002601966FRGMAND100}, value = [ null | 'FRGMAND100' | '000000000002601966' | '0000047504' | '230507' | 'GWM NUDEL-UNIVERSALZWUERZ.1KG' ]).
	at org.apache.kafka.streams.processor.internals.RecordQueue.updateHead(RecordQueue.java:189)
	at org.apache.kafka.streams.processor.internals.RecordQueue.addRawRecords(RecordQueue.java:112)
	at org.apache.kafka.streams.processor.internals.PartitionGroup.addRawRecords(PartitionGroup.java:162)
	at org.apache.kafka.streams.processor.internals.StreamTask.addRecords(StreamTask.java:765)
	at org.apache.kafka.streams.processor.internals.StreamThread.addRecordsToTasks(StreamThread.java:943)
	at org.apache.kafka.streams.processor.internals.StreamThread.runOnce(StreamThread.java:764)
	at org.apache.kafka.streams.processor.internals.StreamThread.runLoop(StreamThread.java:697)
	at org.apache.kafka.streams.processor.internals.StreamThread.run(StreamThread.java:670)
Caused by: java.lang.NullPointerException
	at io.confluent.ksql.execution.streams.timestamp.LongTimestampExtractor.extract(LongTimestampExtractor.java:29)
	at io.confluent.ksql.execution.streams.timestamp.KsqlTimestampExtractor.extract(KsqlTimestampExtractor.java:24)
	at org.apache.kafka.streams.processor.internals.RecordQueue.updateHead(RecordQueue.java:184)
	... 7 more
```

**To Reproduce**
This was found in a customer server log in the Cloud. I don't have the details of the query to reproduce it. However, I could get the version which was 0.8.0

**Expected behavior**
If a timestamp is null, then the row should be ignored and not putting the query in ERROR state.

**Comment by spena** [comment]
The only way I can reproduce a similar error when finding a NULL timestamp is by setting `ksql.timestamp.throw.on.invalid=true`. However, the stacktrace is a little different. The error for this bug says the `RecordQueue.updateHead` called `KsqlTimestampExtractor.extract`. But while debugging the code, I saw that `RecordQueue` should call `LoggingTimestampExtractor` instead. 

I can't find how to make RecordQueue call the internal LoggingTimestampExtractor. Also. the `ksql.timestamp.throw.on.invalid=true` config cannot be set by the CLI. It is a server-only config. I looked in the Cloud instance, but I do not see that config set.
